CHANGELOG.md

```diff
@@ -1,29 +1,37 @@
-# Virtual Attributes Changelog
+# Change Log
 
-Doing our best at supporting [SemVer](http://semver.org/) with
-a nice looking [Changelog](http://keepachangelog.com).
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
```

Comment:
This line should now be changed that we are _NOT_ following symantic version, but instead matching Rails' release version numbers.